src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+
+  &__star {
+    display: flex;
+    margin-right: 4px;
+
+    width: $star-size;
+    height: $star-size;
+
+    background: url(../images/star.svg) center no-repeat;
+  }
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  width: $star-size;
+  height: $star-size;
```

Comment:
- Use nesting
- width and height you already specified for the `&__star` selector
- And the functionality should be the same as in the stars task
<img width="486" alt="image" src="https://github.com/user-attachments/assets/e1d15ba7-e976-410b-84fb-f97044693850">